app/livechat/server/lib/Livechat.js

```diff
@@ -221,7 +221,7 @@ export const Livechat = {
 		return true;
 	},
 
-	registerGuest({ token, name, email, department, phone, username, connectionData } = {}) {
+	registerGuest({ id, token, name, email, department, phone, username, connectionData } = {}) {
 		check(token, String);
 
```

Comment:
Can you check if id is a String? Since this is a public facing endpoint, i don't want ppl to be passing objects as `id`.
`check(id, Match.Maybe(String));`

Reply:
Makes sense, I've added it